UI/MapWnd.cpp

```diff
@@ -1400,20 +1401,31 @@ ModeratorActionSetting MapWnd::GetModeratorActionSetting() const
 bool MapWnd::AutoEndTurnEnabled() const
 { return m_auto_end_turn; }
 
+void MapWnd::PreRender() {
+    GG::Wnd::PreRender();
+    // Save CPU / GPU activity by skipping rendering when it's not needed
+    // As of this writing, the design and research screens have fully opaque backgrounds.
```

Comment:
Only by default, a user may reduce the alpha value in `UI.ctrl-color` or `UI.wnd-color` (both supported in GUI options)